scripts/tag-reconciler/tag-reconciler.go

```diff
@@ -0,0 +1,152 @@
+package main
+
+import (
+	"fmt"
+	"os"
+	"regexp"
+	"strings"
+
+	"github.com/blang/semver/v4"
+	"github.com/sirupsen/logrus"
+	"sigs.k8s.io/release-sdk/git"
+	"sigs.k8s.io/release-utils/env"
+	"sigs.k8s.io/release-utils/util"
+)
+
+const (
+	githubTokenEnvKey = "GITHUB_TOKEN"
+	gitRemoteEnvKey   = "REMOTE"
+	orgEnvKey         = "ORG"
+	versionFile       = "internal/version/version.go"
+	branchPrefix      = "release-"
+	versionPrefix     = "v"
+	crioOrgRepo       = "cri-o"
+)
+
+var releaseMinorVersions = []string{"1.29", "1.28", "1.27"}
```

Comment:
Make this a slice in [version.go](https://github.com/cri-o/cri-o/blob/main/internal/version/version.go) and reuse it in both scripts.